packages/itwin/ec3-widget/public/locales/en-PSEUDO/ec3Widget.json

```diff
@@ -0,0 +1,37 @@
+{
+  "TemplateCreationStepper": {
```

Comment:
What is the use for this pseudo localization? I remember in one of our sprint reviews someone said to NOT use them. Ever.

Reply:
The localization command which already was there in the package.json created it automatically. Should I revert it?

Reply:
Yes. And remove that command.

Reply:
done